runtime/frameworks/legacy/core/array.js

```diff
@@ -1,5 +1,22 @@
 /* eslint-disable */
-
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
 
```

Comment:
Actually, these files are copied from Vue.js v1.0.12 which is using MIT license, so they should not include the Apache 2.0 license comments.

We have already mentioned it in the LICENSE file, please refer to https://github.com/apache/incubator-weex/blob/0.19.0/LICENSE#L213-L242

Reply:
Got it. will reformat the patch. thanks.